docs/helm-chart-checks.md

```diff
@@ -291,11 +291,14 @@ A profile defines a set of checks to run and an indication of whether each check
 - community
   - Defines the requirements for a community chart to pass helm chart certfication.
   - The ```helm-lint``` check is the only mandatory check with all other checks optional.
+- developer-console
```

Comment:
Since we are adding one more profile. Need to modify line 284 `Four profiles are currently available` part.

Reply:
Made the change